Tools/build/stable_abi.py

```diff
@@ -25,6 +25,8 @@
 import csv
 
 SCRIPT_NAME = 'Tools/build/stable_abi.py'
+DEFAULT_MANIFEST_PATH = (
+    Path(__file__).parent / '../../Misc/stable_abi.toml').resolve()
```

Comment:
Worth adding an assertion, so that this breaks in an obvious way if the file is moved?

```suggestion
    Path(__file__).parent / '../../Misc/stable_abi.toml').resolve()
assert DEFAULT_MANIFEST_PATH.exists()
```

Reply:
Oh, the `.resolve()` call already handles that. Don't mind me!

Reply:
It doesn't, it resolves the part that exists and keeps the rest.
But, the point of the argument is that you can run the script with a different file.

That said, I can change Makefile so that a CI check will fail if the default is wrong.

Reply:
Great, thank you!